src/shared/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import Tooltip, { TooltipProps } from './Tooltip'
+import { Text, Button } from '@/shared/components'
+import { Regular as ChannelPreview } from '../ChannelPreview/ChannelPreview.stories'
```

Comment:
I think we shouldn't import from other stories, that way we make them dependent on each other. If we need some component, let's import it directly